src/utils.jl

```diff
@@ -18,6 +23,15 @@ text_only(doc::HTMLDocument) = text_only(doc.root)
 text_only(frag) = join([replace(text(leaf), "\r","") for leaf in Leaves(frag) if leaf isa HTMLText], " ")
 text_only(frags::Vector) = join(text_only.(frags), " ")
 
+function filter_html(random)
```

Comment:
Split this into two methods.
`filter_html(::Void)=nothing`
and `filter_html(content) = ...`
Also `random` is a terrible name, `content` or `text` is better

Reply:
Yes, I was debugging and forgot to update 😅